bitrise.yml

```diff
@@ -1418,6 +1418,44 @@ workflows:
       bitrise.io:
         stack: osx-xcode-14.1.x-ventura
         machine_type_id: g2.8core
+  perfherder-test:
+    steps:
+    - cache-pull@2.1:
+        is_always_run: true
+    - xcode-build-for-test@2:
+        inputs:
+        - test_plan: PerformanceTestPlan
+        - scheme: Fennec_Enterprise_XCUITests
+    - xcode-test@4.5:
+        inputs:
+        - scheme: Fennec_Enterprise_XCUITests
+        - xcodebuild_test_options: "-testPlan PerformanceTestPlan"
+        - simulator_device: "iPhone 14"
+    - script@1.1:
+        title: Create perfherder data object
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+            # debug log
+            set -x
+            # get dependency
+            cd ./test-fixtures && git clone https://github.com/isabelrios/xcresult_extract.git
+            
+            xcrun xcresulttool graph --path $BITRISE_XCODEBUILD_TEST_LOG_PATH/Test-Fennec_Enterprise_XCUITests-*
+            python3 xcresult_extract.py -project ../Client.xcodeproj -scheme Fennec_Enterprise_XCUITests
+            sed 's/.*=//;s/'\''/"/g' data.txt > test.json
+            python3 perfTestTransform.py
+    - deploy-to-bitrise-io:{}
```

Comment:
This line is failing, can you add an space before?
Also, the correct format is (looking at another deploy step above): 
`- deploy-to-bitrise-io@1.9: {}`